ludwig/features/numerical_feature.py

```diff
@@ -265,74 +273,74 @@ def __init__(self, feature):
         logits = inputs[LOGITS]
         predictions = logits
 
         if self.clip is not None:
             if isinstance(self.clip, (list, tuple)) and len(self.clip) == 2:
                 predictions = tf.clip_by_value(
-                    predictions,
-                    self.clip[0],
-                    self.clip[1]
-                )
-                logger.debug(
-                    '  clipped_predictions: {0}'.format(predictions)
+                    predictions, self.clip[0], self.clip[1]
                 )
+                logger.debug("  clipped_predictions: {0}".format(predictions))
             else:
                 raise ValueError(
-                    'The clip parameter of {} is {}. '
-                    'It must be a list or a tuple of length 2.'.format(
-                        self.feature_name,
-                        self.clip
+                    "The clip parameter of {} is {}. "
+                    "It must be a list or a tuple of length 2.".format(
+                        self.feature_name, self.clip
                     )
                 )
 
         return {PREDICTIONS: predictions, LOGITS: logits}
 
     def _setup_loss(self):
-        if self.loss[TYPE] == 'mean_squared_error':
+        if self.loss[TYPE] == "mean_squared_error":
             self.train_loss_function = MSELoss()
-        elif self.loss[TYPE] == 'mean_absolute_error':
+        elif self.loss[TYPE] == "mean_absolute_error":
             self.train_loss_function = MAELoss()
+        elif self.loss[TYPE] == "root_mean_squared_error":
+            self.train_loss_function = RMSELoss()
+        elif self.loss[TYPE] == "root_mean_squared_percentage_error":
+            self.train_loss_function = RMSPELoss()
         else:
             raise ValueError(
-                'Unsupported loss type {}'.format(self.loss[TYPE])
+                "Unsupported loss type {}".format(self.loss[TYPE])
             )
 
         self.eval_loss_function = self.train_loss_function
 
     def _setup_metrics(self):
         self.metric_functions = {}  # needed to shadow class variable
-        if self.loss[TYPE] == 'mean_squared_error':
-            self.metric_functions[LOSS] = MSEMetric(name='eval_loss')
-        else:
-            self.metric_functions[LOSS] = MAEMetric(name='eval_loss')
-        self.metric_functions[ERROR] = ErrorScore(name='metric_error')
+        if self.loss[TYPE] == "mean_squared_error":
+            self.metric_functions[LOSS] = MSEMetric(name="eval_loss")
+        elif self.loss[TYPE] == "mean_absolute_error":
+            self.metric_functions[LOSS] = MAEMetric(name="eval_loss")
+        elif self.loss[TYPE] == "root_mean_squared_error":
+            self.metric_functions[LOSS] = RMSEMetric(name="eval_loss")
+        elif self.loss[TYPE] == "root_mean_squared_percentage_error":
+            self.metric_functions[LOSS] = RMSPEMetric(name="eval_loss")
+
+        self.metric_functions[ERROR] = ErrorScore(name="metric_error")
```

Comment:
i think we can get rid of error, it's just not informative. can also get rid of ErrorScore.

Reply:
@w4nderlust see [commit](c446be85bdc9755816985bd56b0a0a23ea14e867)